src/userConfig_sample.h

```diff
@@ -63,6 +63,7 @@ enum MACHINE {
 #define PINMODEVOLTAGESENSOR INPUT // Mode INPUT_PULLUP, INPUT or INPUT_PULLDOWN_16 (Only Pin 16)
 #define PRESSURESENSOR 0           // 0 = no pressure sensor connected, 1 = pressure sensor connected
 #define TEMP_LED 1                 // Blink status LED when temp is in range
+#define WATER_SENS 0               // 0 = no water sensor, 1 = water sensor XKC-Y25-NPN connected
```

Comment:
see comment on digital read, config for npn or pnp missing

Reply:
Could be defined like the switches 0 =no sensor 1= npn 2 = pnp

Reply:
Do you have a PNP to test? I have only NPN...

Reply:
OK implemented